core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BillingRepository.scala

```diff
@@ -45,4 +46,15 @@ class BillingRepository(dataSource: SlickDataSource) {
     dataSource.inTransaction { dataAccess =>
       dataAccess.rawlsBillingProjectQuery.delete(projectName)
     }
+
+  def updateCreationStatus(projectName: RawlsBillingProjectName, status: CreationStatus, message: Option[String]): Future[Int] = {
```

Comment:
I ended up with a nearly identical implementation for this on my branch. The only difference is I called it `updateStatus` instead.
Realistically, I think I'll need to merge in your changes before doing anything productive with the polling for landing zone completion anyway, so I'll just drop my changes when I merge.